src/pages/lesson-plans/lesson-06.html

```diff
@@ -134,7 +134,7 @@
     </HEAD>
     <BODY>
     <div class="lesson-plan-wrapper">
-      <div class="lesson-plan">
+      <lesson-plan [lessonNumber]="6">
```

Comment:
Here is the start of a LessonPlan component -- it's just a draft.  It may not actually be necessary to pass in the lesson number.  You might need other things here in addition.  Whatever.